example/test/exampleFluentDslLabelDefinitions.js

```diff
@@ -1,18 +1,21 @@
+// when using the npm module, use `import {fluentDslConfig} from 'scenarioo'` instead
+import {fluentDslConfig} from '../../lib';
```

Comment:
does not work when running our examples (no support for this module import in pure javascript here)